amethyst_renderer/src/pass/shaders/geometry/debug_lines.glsl

```diff
@@ -0,0 +1,62 @@
+// TODO: Needs documentation.
+
+#version 150 core
+
+layout (std140) uniform VertexArgs {
+    uniform mat4 proj;
+    uniform mat4 view;
+    uniform mat4 model; /* Not actually used, all our lines are in world coordinates */
+};
+
+in VertexData {
+    vec3 position;
+    vec4 color;
+    vec3 normal;
+} vertex_in[1];
+
+out VertexData {
+    vec3 position;
+    vec4 color;
+    vec3 normal;
+} vertex;
+
+layout (points) in;
+layout (triangle_strip, max_vertices = 4) out;
+
+uniform vec3 camera_position;
+
+const float WIDTH = 2.0 / 800.0;
```

Comment:
Where's the 800 from ?

Reply:
It was meant to be `1.0 / 400.0`, as this way is easy to set the line width relative to the unit value `1.0`. As in `the line width is one line width unit, which is one four-hundredth the unit size`. This also makes it easier to use intuitive values for the line width instead of small values like 0.0001, 0.015, 0.003... As simple using line width 1.0 will result in in way too thick lines. 

The value `2.0 / 800.0` was chosen arbitrarily as it didn't look too narrow nor too thick.
I will add a comment right above to make it clearer.

Reply:
Thanks